edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Core.Test/TestRuntimeModule.cs

```diff
@@ -22,10 +22,10 @@ public class TestRuntimeModule : TestModule, IRuntimeModule<TestConfig>
             DateTime lastRestartTimeUtc,
             ModuleStatus runtimeStatus,
             ImagePullPolicy imagePullPolicy = ImagePullPolicy.OnCreate,
-            uint priority = Constants.DefaultPriority,
+            uint startupOrder = Constants.DefaultPriority,
```

Comment:
better to rename Constants.DefaultPriority.

Reply:
That's done in 1.0.10 I believe.

Reply:
No, it still shows DefaultPriority in 1.0.10 branch, https://github.com/Azure/iotedge/blob/70661641799d16a940b191ee1c7faa24842fd4be/edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Core.Test/TestRuntimeModule.cs#L25.